homework-3/ProjectSIIT/src/main/java/ro/siit/java10/evp/ro/siit/java10/evp/service/CarService.java

```diff
@@ -0,0 +1,55 @@
+package ro.siit.java10.evp.ro.siit.java10.evp.service;
+
+import ro.siit.java10.evp.Car;
+import ro.siit.java10.evp.DealerCar;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.lang.Boolean.TRUE;
+
+public class CarService {
+
+    /**
+     * Filter list of {@Link DealerCar} by fastCharging, if is true or false
+     *
+     * @param dealerCars
+     * @param fastCharging
+     * @return a new list filtered by fast charging
+     */
+    public List<DealerCar> filterByFastCharging(List<DealerCar> dealerCars, boolean fastCharging) {
```

Comment:
dealerCars should be something this class has access to and not something you pass there